ci/build-helpers/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+import org.jetbrains.compose.internal.publishing.*
+
+plugins {
+    signing
+}
+
+val mavenCentral = MavenCentralProperties(project)
```

Comment:
This file is copied from https://github.com/JetBrains/compose-multiplatform-core/blob/2d8c5f479f590305b9fedc744e00fdc9c276bbb9/mpp/build.gradle.kts#L271